sympy/polys/tests/test_numberfields.py

```diff
@@ -545,6 +545,7 @@ def test_AlgebraicNumber():
     assert a.is_aliased is True
 
     assert AlgebraicNumber(sqrt(2), []).rep == DMP([], QQ)
+    assert AlgebraicNumber(sqrt(2), (0, 0)).rep == DMP([], QQ)
```

Comment:
Actually, it was ``_print_AlgebraicNumber`` that was broken. And there are tests for that.